packages/vats/test/upgrading/test-upgrade-contracts.js

```diff
@@ -32,7 +32,7 @@ test('upgrade mintHolder', async t => {
 
   /** @type {SwingSetConfig} */
   const config = harden({
-    defaultManagerType: 'xs-worker',
+    defaultManagerType: 'local', // Overridden in CI with SWINGSET_WORKER_TYPE=xs-worker
```

Comment:
relying on an environment variable to override this seems like spooky-action-at-a-distance.

But... I don't suppose I have a better idea.

Reply:
That's the way all the other `test:xs` scripts are structured, so at least there's a precedent.